impacted by github actions upstream failure for claude code

https://github.com/anthropics/claude-code-action/issues/1205

https://github.com/kratsg/giordonstark.com/actions/runs/24321357807 fails with

```
Run anthropics/claude-code-action@v1
Run oven-sh/setup-bun@3d267786b128fe76c2f16a390aa2448b815359f3
Downloading a new version of Bun: https://github.com/oven-sh/bun/releases/download/bun-v1.3.6/bun-linux-x64.zip
/usr/bin/unzip -o -q /home/runner/work/_temp/10c3ddb0-ab7e-483b-a2d6-742cbba1bdbc.zip
/home/runner/.bun/bin/bun --revision
1.3.6+d530ed993
Run cd ${GITHUB_ACTION_PATH}
bun install v1.3.6 (d530ed99)

+ @actions/core@1.11.1
+ @actions/github@6.0.1
+ @anthropic-ai/claude-agent-sdk@0.2.101
+ @modelcontextprotocol/sdk@1.16.0
+ @octokit/graphql@8.2.2
+ @octokit/rest@21.1.1
+ @octokit/webhooks-types@7.6.1
+ node-fetch@3.3.2
+ shell-quote@1.8.3
+ zod@3.25.76

163 packages installed [2.09s]
Run bun --no-env-file \
Issue title contains exact trigger phrase '@claude'
Auto-detected mode: tag for event: issues
Requesting OIDC token...
Attempt 1 of 3...
OIDC token successfully obtained
Exchanging OIDC token for app token...
Attempt 1 of 3...
App token exchange failed: 401 Unauthorized - Failed to get GitHub App installation token
Attempt 1 failed: Failed to get GitHub App installation token
Retrying in 5 seconds...
Attempt 2 of 3...
App token exchange failed: 401 Unauthorized - Failed to get GitHub App installation token
Attempt 2 failed: Failed to get GitHub App installation token
Retrying in 10 seconds...
Attempt 3 of 3...
App token exchange failed: 401 Unauthorized - Failed to get GitHub App installation token
Attempt 3 failed: Failed to get GitHub App installation token
Operation failed after 3 attempts
Error: Action failed with error: Failed to get GitHub App installation token
Internal error: directory mismatch for directory "/home/runner/work/_actions/anthropics/claude-code-action/v1/tsconfig.json", fd 4. You don't need to do anything, but this indicates a bug.
Error: Process completed with exit code 1.
Run BUN_BIN="${GITHUB_ACTION_PATH}/bin/bun"
No buffered inline comments
Internal error: directory mismatch for directory "/home/runner/work/_actions/anthropics/claude-code-action/v1/tsconfig.json", fd 4. You don't need to do anything, but this indicates a bug.
```

--- a/.github/workflows/claude.yml
+++ b/.github/workflows/claude.yml
@@ -41,7 +41,7 @@ jobs:
 
       - name: Run Claude Code
         id: claude
-        uses: anthropics/claude-code-action@v1
+        uses: anthropics/claude-code-action@232c9a15f49822c165d462e7f0a3aa9287ed2a60
         with:
           claude_code_oauth_token: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN }}
           additional_permissions: |
@@ -77,7 +77,7 @@ jobs:
 
       - name: Run Claude Code
         id: claude
-        uses: anthropics/claude-code-action@v1
+        uses: anthropics/claude-code-action@232c9a15f49822c165d462e7f0a3aa9287ed2a60
         with:
           claude_code_oauth_token: ${{ secrets.CLAUDE_CODE_OAUTH_TOKEN }}
           additional_permissions: |

